lib/workers/helpers.rb

```diff
@@ -4,7 +4,7 @@
 module WorkerHelpers
 
   def log_info(message)
-    logger.info(message) if ::RubyDeps.enable_logging
+    logger.info(message) if defined?(RubyDeps) && ::RubyDeps.enable_logging
```

Comment:
When would RubyDeps not be defined?

Reply:
I think I had this problem when setting up the tests, but I think that should work now!